applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c

```diff
@@ -140,3 +140,71 @@ int le_audio_stream_dir_get(struct bt_bap_stream const *const stream)
 
 	return ep_info.dir;
 }
+
+void le_audio_print_codec(const struct bt_audio_codec_cfg *codec, enum bt_audio_dir dir)
```

Comment:
Not very necessary, we can handle this in another PR: Could this API also be reused in broadcast sink?

Reply:
I did look into it, but it requires quite a bit of rewrite, so we should do it in a separate PR